plugins/woocommerce/src/Internal/Caches/ProductCacheController.php

```
@@ -0,0 +1,109 @@
+		$this->product_cache->remove( $post_id );
+	}
+
+	/**
+	 * Invalidate the product cache when post meta is updated.
+	 *
+	 * @param int $meta_id   The ID of the metadata entry.
+	 * @param int $object_id The ID of the object the metadata is for.
+	 *
+	 * @return void
+	 */
+	public function maybe_invalidate_product_cache_by_meta( $meta_id, $object_id ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$meta_id'. (undefined)

(UnusedFormalParameter)